### DISCUSSION TITLE
Rendering Engine 1 or 2 for Lenovo M10 Tablet (3rd gen.)?

### DISCUSSION THREAD

**Discussion by JensKlaus** [osmandapp/OsmAnd#18442] [Maps Related]
I like to use the slope map function in OSMAND. It works fine with Version 1 of the map rendering engine.

However, when I switch to Version 2 (OpenGL) I only get a flat map without any relief shading and without slope coloring. Is the hardware (Lenovo M10 Tablet 3rd Gen with Android 12) too old for Version 2 or do I have to change any additional settings?

**Comment by DmitryAlexei** [comment]
In order to use slopes with rendering version 2 you need to download new Terrain maps (3d) using Maps and Resources menu 

You can also simply enable *Use raster sqlitedb for hillshade / slope * option in the development plugin (go to menu > plugins > development plugin > settings ). In that case, old raster maps will be used to display slopes (and new maps will not) with rendering version 2

**Comment by JensKlaus** [comment]
Thanks, DimitryAlexei.

The work around with the developer plugin works fine. 

Unfortunately, way 1 using the new 3d maps and engine 2 (openGL) still doesn't show any part of the relief or colored hill slopes.

**Reply by DmitryAlexei** [reply]
@JensKlaus there was an issue with hillshades in 4.5.8 version of the app (https://github.com/osmandapp/OsmAnd/issues/17783). You can try to download the latest version from Google play (or try beta)  and see if fix is working for you

**Reply by JensKlaus** [reply]
Thanks for your information. However, I run Osmand 4.5.10 with that issue.

**Reply by DmitryAlexei** [reply]
This bug is completely fixed in the latest beta builds (https://github.com/osmandapp/OsmAnd/issues/18179#issuecomment-1735602635).
